pkg/snmp/traps/status/status.go

```diff
@@ -54,8 +86,8 @@ func GetStatus() map[string]interface{} {
 	}
 	status["metrics"] = metrics
 
-	if startError != nil {
-		status["error"] = startError.Error()
-	}
+	// if startError != nil {
+	// 	status["error"] = startError.Error()
+	// }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why this is commented?

Reply:
Oops, I meant to remove that entirely. Once this is componentized, the error is handled as part of the app startup, so we won't have access to it.
Removed.